app/controllers/games_controller.rb

```diff
@@ -1,10 +1,13 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!
+
   def new
     @game = Game.new
   end
 
   def create
     @game = Game.create(game_params)
+    @game.players.create(game_id: @game.id, user_id: current_user.id, color: [:black, :white].sample)
```

Comment:
Just curious, since we defined that a player belongs to a game and a player, couldn't we write something like @game.players.create(user: current_user, color: [:black, :white].sample) ?